_tools/lint_plugin.ts

```diff
@@ -38,5 +43,67 @@ export default {
         };
       },
     },
+    // https://docs.deno.com/runtime/contributing/style_guide/#error-messages
+    "error-message": {
+      create(context) {
+        return {
+          NewExpression(node) {
+            if (node.callee.type !== "Identifier") return;
+            const name = node.callee.name;
+            if (!name.endsWith("Error")) return;
+            const argument = node.arguments[0];
+            if (argument?.type !== "Literal") return;
+            const value = argument.value;
+            if (typeof value !== "string") return;
+
+            if (value.match(LEADING_LOWERCASE_REGEXP)) {
```

Comment:
```suggestion
            if (value[0] === value[0].toLowerCase()) {
```

Reply:
This would return a false positive if it is not a letter char e.g. `"@" === "@".toLowerCase()`

Reply:
We could to `value[0] !== value[0].toUpperCase()` but I don't know if this would bring unintentional behaviors for special chars. WDYT?

Reply:
Try that 🤷🏾